presto-postgresql/src/main/java/io/prestosql/plugin/postgresql/PostgreSqlClient.java

```diff
@@ -138,14 +138,19 @@ protected ResultSet getTables(Connection connection, Optional<String> schemaName
     @Override
     public Optional<ColumnMapping> toPrestoType(ConnectorSession session, JdbcTypeHandle typeHandle)
     {
-        switch (typeHandle.getJdbcTypeName()) {
+        if (!typeHandle.getJdbcTypeName().isPresent()) {
+            return Optional.empty();
```

Comment:
This should go to `super.toPrestoType`.
However, since we know postgresql types are properly named we can simply

```
String jdbcTypeName = typeHandle.getJdbcTypeName().orElseThrow
```

then you don't need `if (!typeHandle.getJdbcTypeName().isPresent())` at all

Reply:
`super.toPrestoType` does not use `getJdbcTypeName`